packages/cozy-harvest-lib/src/components/OAuthForm.jsx

```diff
@@ -59,9 +60,13 @@ export const OAuthForm = props => {
     if (typeof onSuccess === 'function') onSuccess(accountId)
   }
 
-  const handleConnect = () => {
+  const handleConnect = useCallback(() => {
     setShowOAuthWindow(true)
-  }
+    const konnectorPolicy = findKonnectorPolicy(konnector)
+    if (konnectorPolicy.isBIWebView) {
+      flow.expectTriggerLaunch({ konnector })
+    }
+  }, [client, flow, konnector.slug])
```

Comment:
Why `konnector.slug`? You are using the whole object `konnector`. So this should be this object within the dep array. 

Your IDE should have tell you that.